src/coins.h

```diff
@@ -212,6 +212,9 @@ class CCoinsViewCache : public CCoinsViewBacked
     uint256 GetBestBlock() const;
     void SetBestBlock(const uint256 &hashBlock);
     bool BatchWrite(CCoinsMap &mapCoins, const uint256 &hashBlock);
+    CCoinsViewCursor* Cursor() const override {
```

Comment:
This 'override' is causing tons of warnings here.
```
/home/orion/projects/bitcoin/bitcoin/src/coins.h:210:10: warning: 'GetCoin' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
    bool GetCoin(const COutPoint &outpoint, Coin &coin) const;
```
Some compilers (clang5, in this case) demand that a whole class is annotated with it if it is used anywhere. So I'd argue against adding it here, or in a separate commit add it to all appropriate CCoinsViewCache methods.